### ISSUE TITLE
Womens vignette failing travis build

### ISSUE THREAD

**Issue by jimmyday12** [jimmyday12/fitzRoy#52]
The travis check is failing R CMD CHECK for the womens vignette. The following error message is received 

```
Quitting from lines 35-36 (womens-stats.Rmd) 
Error: processing vignette 'womens-stats.Rmd' failed with diagnostics:
Evaluation error: is.character(x) is not TRUE.
```

The build passes all checks for me locally and I can knit the vignette fine locally so I'm unsure what is going on. For now, I've prevented the vignette from executing code to see if that helps.

**Comment by jimmyday12** [comment]
@OscarLane Just tagging you in as you wrote the vignette so might have some thoughts. I'll see if I can do some digging

**Comment by jimmyday12** [comment]
Seemed to have fixed this in the commit a8902ce. Was related to a line of code in get_aflw_round_data() that was using readr::parse_datetime. I've commented that line out and it seems to work. Also has fixes #53!
